### PR TITLE
Feat: add MessageCollection

### PR DESCRIPTION
**Pull request by T0ffik**

### CODE REVIEW

File: src/components/organisms/MessageCollection/MessageCollection.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { MessageComponent } from '../..';
+
+export const MessageCollection = () => {
+    const usersMsgs = [
```
Comment (Veryyapeee) [inline src/components/organisms/MessageCollection/MessageCollection.jsx:4]: Create separate file for mocks, just something like Mocks.js file

File: src/components/organisms/MessageCollection/MessageCollection.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        {
+            profileId: '3',
+            name: 'Mariusz',
+            isYour: false,
+            date: new Date(),
+            message: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Ea eius necessitatibus accusantium vero a maiores nobis fugit provident saepe, reiciendis sit assumenda iusto facilis voluptates cumque reprehenderit vel qui eaque?'
+        },
+    ];
+
+    return (usersMsgs.map((user) => {
```
Comment (Veryyapeee) [inline src/components/organisms/MessageCollection/MessageCollection.jsx:29]: just
`usersMsgs.map(user => (<MessageComponent/>))`

File: src/pages/Chat/Chat.jsx [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import { MessageCollection } from '../../components';
+
 export const ChatPage = () => {
-    return <div>This is chat page</div>;
+    return <MessageCollection />;
```
Comment (Veryyapeee) [inline src/pages/Chat/Chat.jsx:4]: actually the data should be passed to MessageCollection, we won't have access to data directly in message collection, we would have access to them in screen

File: src/components/organisms/MessageCollection/MessageCollection.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        return (
+            <MessageComponent
+                date={user.date}
+                name={user.name}
```
Comment (Veryyapeee) [inline src/components/organisms/MessageCollection/MessageCollection.jsx:33]: Missing key prop, just add ID field to mock (we would get it from API anyway), and use it as a key, key has to be unique

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/components/organisms/MessageCollection/MessageCollection.jsx (hunk 1 above)
Comment (T0ffik) [reply src/components/organisms/MessageCollection/MessageCollection.jsx:4]: Done

File: src/components/organisms/MessageCollection/MessageCollection.jsx (hunk 2 above)
Comment (T0ffik) [reply src/components/organisms/MessageCollection/MessageCollection.jsx:29]: Done

File: src/pages/Chat/Chat.jsx (hunk 3 above)
Comment (T0ffik) [reply src/pages/Chat/Chat.jsx:4]: Done

File: src/components/organisms/MessageCollection/MessageCollection.jsx (hunk 4 above)
Comment (T0ffik) [reply src/components/organisms/MessageCollection/MessageCollection.jsx:33]: Done

**Review by Mariusz-web — APPROVED** [review]

**Pull request merged**